src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -20,6 +23,8 @@
  */
 public class FilterCommandParser implements Parser<FilterCommand> {
 
+    public static final String MESSAGE_SUPPORT = "filter does not support Name, Email or Phone\n"
```

Comment:
Should we capitalize the first letter of each line here so it looks neater in the result display?